hack/snapshot_to_catalog.sh

```diff
@@ -1,23 +1,24 @@
 #!/usr/bin/env bash
 
 usage() {
-  echo "Usage: $0 -s <snapshot-ref> -c <catalog-file>"
-  echo "  snapshot-ref: required, the snapshot reference to use, example: ols-bnxm2"
+  echo "Usage: $0 -s <snapshot-refs> -c <catalog-file>"
+  echo "  snapshot-refs: required, the snapshots' references to use, ordered by versions ascending, example: ols-cq8sl,ols-mdc8x"
```

Comment:
@raptorsun We dont have a mechanism to keep the snapshots for a long time . How/When will we use the 2 snapshot situation ?

Reply:
for the case we want to keep 2 versions in the same channel. 
At the moment all our versions in an upgrade path leading directly to the latest version. If we have a breaking upgrade that must pass by an intermediate version, it will be helpful.

Reply:
@raptorsun Do we need to update the konflux snapshot expiration ? If we dont have the snapshot live long enough we cannot use this multi snapshot version at all .

Reply:
the catalog is refering to the production images instead of short live quay.io images https://github.com/openshift/lightspeed-operator/pull/381/files#diff-39d45c384da54b53ec1276626d0ad3790c04e195fd5980ec841d56f07d3524b5R49-R57

Reply:
correct - but we want to be able to see the old snapshot in konflux `oc get snapshot` should list , or the script will not know about the Digest it had.